backend/src/plugins/Automod/functions/matchMultipleTextTypesOnMessage.ts

```diff
@@ -59,4 +60,8 @@ export async function* matchMultipleTextTypesOnMessage(
       break;
     }
   }
+
+  if (trigger.match_polls && msg.data.poll) {
+    yield ["poll", JSON.stringify(msg.data.poll)];
```

Comment:
This will not work for the `answers` prop, since `<Collection>.toJSON()` returns an array with the keys only (for whatever reason...)

You're going to have to construct an object with the desireable keys instead, and manually call `answers.map(...)` or `Object.fromEntries(answers.entries())`, depending on the format you want, and then call JSON.stringify on that.

Reply:
The hell...? Thanks for that though I'll fix it

Reply:
Nevermid, this is a SavedMessage so the `answers` prop is an array, not a Collection